### PR TITLE
Adds tabs to navbar

### PR DESCRIPTION
**Pull request by valeriaconde**

### CODE REVIEW

File: src/components/Menu.js [hunk 1]
```diff
@@ -11,9 +11,13 @@ class Menu extends Component {
             <Navbar bg="light" variant="light">
                 <Navbar.Brand href="#home"><Image src="/logo.png" height={"35"} /></Navbar.Brand>
                 <Nav className="mr-auto">
-                    <Nav.Link href="#home">Proyectos</Nav.Link>
-                    <Nav.Link href="#clientes">Clientes</Nav.Link>
+                    <Nav.Link href="#home">Clientes</Nav.Link>
+                    <Nav.Link href="#clientes">Usuarios</Nav.Link>
+                    <Nav.Link href="#reportes">Proyectos</Nav.Link>
```
Comment (EliasMera) [inline src/components/Menu.js]: I don't think all these would be #reportes

**Review by EliasMera — CHANGES_REQUESTED** [review]
Please address comments

File: src/components/Menu.js (hunk 1 above)
Comment (valeriaconde) [reply src/components/Menu.js]: True

**Review by EliasMera — APPROVED** [review]
Looks good 🚢 it!

**Pull request merged**